prac_07/guitar.py

```diff
@@ -0,0 +1,11 @@
+class Guitar:
+    def __init__(self, name, year, cost):
+        self.name = name
```

Comment:
__init__(self, name, year, cost): This is the constructor method for the Guitar class. It initializes a new instance of the class with the provided name, year, and cost attributes